### PR TITLE
Fix: Incorrect suicide setup of GLA Demo Stinger Site

### PR DESCRIPTION
**Pull request by xezon**
This change fixes the incorrect suicide setup of the GLA Demo Stinger Site.

### CODE REVIEW

**Comment by xezon** [comment]
The `Demo_DestroyedWeapon` does not work on Stinger Site.

**Comment by xezon** [comment]
Documentation added. Looks good.

File: Patch104pZH/Design/Changes/v1.0/2308_demo_stinger_death_damage.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+date: 2023-08-29
+
+title: Fixes death explosion damage of upgraded GLA Demo Stinger Site
+
+changes:
+  - fix: The upgraded GLA Demo Stinger Site no longer deals 20 times more damage than it should when killed.
+  - fix: The upgraded GLA Demo Stinger Site now properly deals damage when killed by poison.
```
Comment (commy2) [inline Patch104pZH/Design/Changes/v1.0/2308_demo_stinger_death_damage.yaml:8]: Sorry for pointing this out so late, but SwallowDamageTypesIfSlavesNotExisting ensures that Stinger Site does not take damage from POISON damage, and all POISONED_X death type weapons deal POISON damage. The fix is good, but the effect is theoretical.

Comment (xezon) [reply Patch104pZH/Design/Changes/v1.0/2308_demo_stinger_death_damage.yaml:8]: Fixed.

**Pull request merged**